app/views/content/landing/careers.md

```diff
@@ -6,6 +6,7 @@ content:
     - content/landing/careers/collage
     - content/landing/careers/mailing_list
     - content/landing/careers/content
+    - content/shared/block-promos/mailing_adviser_routes
```

Comment:
This isn't going to work for this landing page unfortunately - it is for 16 to 18 year olds, so they are not eligible for an adviser

In addition, we have to have a clear message about being 16 years or over for the mailing list

So I think we need a new promo block (sorry!) - probably mailing list with bespoke text and routes??